### PR TITLE
Upgrade to the latest mongoose and new CI

### PR DESCRIPTION
**Pull request by akurilov**
- Use GL CI instead of Travis: https://gitlab.com/emc-mongoose/mongoose-storage-driver-pravega/pipelines
- GL CI allows automated release on the `latest` tag, artifacts preserving (useful for test logs), artifacts publishing (test images are not being pushed to dockerhub)
- Unfortunately, the tests using docker are not working correctly under GL CI yet. We should manually execute such tests (integration & functional)
- Use mongoose-base-4.2.3, mongoose-storage-driver-coop-4.2.3
- Java 11 and YAML configs

### CODE REVIEW

File: ci/.gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    - ./gradlew test
+  artifacts:
+    paths:
+      - build/reports/tests/test/*
+    when: on_failure
+
+test_integration:
+  stage: test
+  services:
+    - name: pravega/pravega:0.3.2
```
Comment (akurilov) [inline ci/.gitlab-ci.yml]: declaring the pravega node service here instead of the test code

File: ci/.gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    - export VERSION=$(cat build.gradle | grep version\ = | sed -n 's/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+    - docker save ${IMAGE_NAME}:${VERSION} > ${IMAGE_FILE_NAME}
+  artifacts:
+    paths:
+      - ${IMAGE_FILE_NAME}
+
+.test_robot:
+  stage: test_functional
+  services:
+    - name: pravega/pravega:0.3.2
```
Comment (akurilov) [inline ci/.gitlab-ci.yml]: declaring the pravega node service here instead of the test code

File: README.md [hunk 3]
```diff
@@ -119,7 +126,7 @@ docker run \
 | storage-driver-scaling-factor     | integer         | 0             | The scaling policy factor. From the Pravega javadoc: *the maximum number of splits of a segment for a scale-up event.*
 | storage-driver-scaling-segments   | integer         | 1             | From the Pravega javadoc: *the minimum number of segments that a stream can have independent of the number of scale down events.*
 | storage-net-node-addrs            | list of strings | 127.0.0.1     | The list of the Pravega storage nodes to use for the load
-| storage-net-node-port             | integer         | 9020          | The default port of the Pravega storage nodes, should be explicitly set to 9090 (the value used by Pravega by default)
+| storage-net-node-port             | integer         | ?             | The default port of the Pravega storage nodes, should be explicitly set to 9090 (the value used by Pravega by default)
```
Comment (veronikaKochugova) [inline README.md]: What for "?" instead of "9020"?

**Review by veronikaKochugova — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (akurilov) [reply README.md]: 9090

**Pull request merged**